client/index.html

```diff
@@ -380,6 +379,18 @@ <h1 class="title">Settings</h1>
 								<textarea class="input" name="userStyles" id="user-specified-css-input" placeholder="You can override any style with CSS here"></textarea>
 							</div>
 						</div>
+
+						{{#unless public}}
+							<div class="session-list">
+								<h2>Sessions</h2>
+								
+								<h3>Current session</h3>
+								<div id="session-current"></div>
+								
+								<h3>Other sessions</h3>
+								<div id="session-list"></div>
```

Comment:
At the moment, this section is empty when there is no other known session, yet the title is still present. What about the following?

<img width="475" alt="screen shot 2017-09-23 at 20 44 23" src="https://user-images.githubusercontent.com/113730/30778300-1b51f2ee-a0a0-11e7-8a58-fe4ecc1b97ac.png">

Achieved by:

```diff
─────────────────────────
modified: client/index.html
─────────────────────────
@ index.html:386 @
                        {{#unless public}}
                            <div class="session-list">
                                <h2>Sessions</h2>
-
+
                                <h3>Current session</h3>
                                <div id="session-current"></div>
-
+
                                <h3>Other sessions</h3>
-                               <div id="session-list"></div>
+                               <div id="session-list">
+                                   <p>
+                                       <em>You are not currently logged in to any other device.</em>
+                                   </p>
+                               </div>
                            </div>
                        {{/unless}}
                    </div>
─────────────────────────
modified: client/js/lounge.js
─────────────────────────
@ lounge.js:409 @ $(function() {
        }

        if (type === "settings") {
-           $("#session-list").text("Loading…");
            socket.emit("sessions:get");
        }

─────────────────────────
modified: client/js/socket-events/sessions_list.js
─────────────────────────
@ sessions_list.js:20 @ socket.on("sessions:list", function(data) {
        html += templates.session(connection);
    });

-   $("#session-list").html(html);
+   if (html.length > 0) {
+       $("#session-list").html(html);
+   }
```

(I'm not pushing for removal of `Loading…`, it was just done quickly to not have it delete the added placeholder)